internal/restore_point_list_handler.go

```diff
@@ -0,0 +1,36 @@
+package internal
```

Comment:
Restore point logic is greenplum-specific and should belong to the `internal/databases/greenplum`.